typings/index.d.ts

```diff
@@ -89,6 +89,10 @@ declare module 'discord.js' {
     public valueOf(): string;
   }
 
+  export class BaseEmoji extends Emoji {
```

Comment:
`E` comes after `C`

Reply:
Oops, I apologize, I've fixed it now :+1: